src/UnknownUrlHandler.cpp

```diff
@@ -24,11 +24,11 @@ QNetworkReply* UnknownUrlHandler::handleRequest(
         QTextStream(stderr) <<
            "Request to unknown URL: " << url.toString() << endl <<
            "To block requests to unknown URLs:" << endl <<
-           "  page.driver.block_unknown_urls" << endl <<
+           "  config.block_unknown_urls" << endl <<
```

Comment:
Perhaps we should be clear where `config` comes from? Maybe wrap this in `Capybara::Webkit.configure { |config| }`?